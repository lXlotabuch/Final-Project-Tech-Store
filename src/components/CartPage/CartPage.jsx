import React, { useEffect } from 'react';
import Heading from '../common/Heading/Heading';
import {ContainerCart} from '../common/Container';
import CartList from './CartList/CartList';
import Summary from './Summary/Summary';
import { RowRevers } from './Flex';

const CartPage = () => {
  useEffect(() => {
    window.scrollTo(0, 0)
  }, [])
  
  return (
    <div>
      <ContainerCart>
        <Heading>
          Shopping Cart
        </Heading>
        <RowRevers>
          <Summary />
          <CartList />
        </RowRevers>
      </ContainerCart>
    </div>
  )
}

export default CartPage;