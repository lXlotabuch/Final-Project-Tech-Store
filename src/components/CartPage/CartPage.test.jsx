import React from 'react'
import {render} from '@testing-library/react'
import CartPage from './CartPage'

jest.mock('./Summary/Summary', () => 'summary')
jest.mock('./CartList/CartList', () => 'cart-list')

test('render CartPage component', () => {
  const {asFragment} = render(<CartPage />)

  expect(asFragment()).toMatchSnapshot()
})