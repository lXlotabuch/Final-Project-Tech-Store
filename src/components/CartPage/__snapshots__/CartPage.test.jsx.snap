// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render CartPage component 1`] = `
<DocumentFragment>
  <div>
    <div
      class="sc-gsTCUz sc-dlfnbm dcszxA gCucIE"
    >
      <h3
        class="sc-bdfBwQ cbmVJ"
      >
        Shopping Cart
      </h3>
      <div
        class="sc-hKgILt sc-jSgupP bGMZSh irRWBl"
      >
        <summary />
        <cart-list />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
